examples/import-database-flyway-maven/README.adoc

```diff
@@ -0,0 +1,114 @@
+= Import Database By Flyway By Maven Plugin
+:index-group: Import
+:jbake-type: page
+:jbake-status: not published/unrevised
+
+This is an example of how to use the Flyway tool to import a database via the Maven Plugin.
```

Comment:
`a database schema and related data via the Flyway Maven Plugin`

Reply:
So, from what I understand, I should change the sentence, and it should look like this:
"This is an example of how to use a database schema and related data via the Flyway Maven Plugin."

Reply:
Yes